gender_analysis/analysis/gender_frequency.py

```diff
@@ -264,10 +264,10 @@ def subject_vs_object_pronoun_freqs(corp, pickle_filepath_male=None, pickle_file
     :return: tuple of two dictionaries (male, female)
 
     >>> from gender_analysis.corpus import Corpus
-    >>> from gender_analysis.common import TEST_DATA_PATH
-    >>> filepath = TEST_DATA_PATH / 'test_corpus'
-    >>> csvpath = TEST_DATA_PATH / 'test_corpus' / 'test_corpus.csv'
-    >>> subject_vs_object_pronoun_freqs(Corpus(filepath, csv_path=csvpath))
+    >>> from gender_analysis.testing.common import TEST_CORPUS_PATH, SMALL_TEST_CORPUS_CSV
+    >>> filepath = TEST_CORPUS_PATH
+    >>> csvpath = SMALL_TEST_CORPUS_CSV
```

Comment:
In order to keep things concise, could you use an `import _ as _` statement? Ultimately it's not that big of a deal, but will probably make it more clear for people as they read the docs.

Reply:
Done!